fsd_config/form_jsons/cof_r3/en/operational-costs-cof-r3-w1.json

```diff
@@ -97,7 +98,7 @@
             "columnTitles": ["Source", "Amount", "Action"]
           },
           "type": "MultiInputField",
-          "title": "Your income sources",
+          "title": "Sources of income",
```

Comment:
@adamdavies1 Just to double check, will this render as "Sources of income" **only** in the Check your answers page?

Reply:
Yes, thats right